bridge-deploy/Makefile

```diff
@@ -0,0 +1,41 @@
+VIRTUAL_ENV ?= venv
+
+lint: install-requirements
+	$(VIRTUAL_ENV)/bin/flake8 bridge_deploy tests setup.py
+	$(VIRTUAL_ENV)/bin/black --check bridge_deploy tests setup.py
+	$(VIRTUAL_ENV)/bin/mypy bridge_deploy tests setup.py --ignore-missing-imports
+
+format:
+	$(VIRTUAL_ENV)/bin/black bridge_deploy tests setup.py
+
+test: install-requirements compile
+	$(VIRTUAL_ENV)/bin/pytest -s tests
```

Comment:
Should remove the `-s` otherwise pytest will print confusing stuff from non failing tests. (afaik)

Reply:
Debugging leftover ;-)